components/scream/src/share/field/field_manager.hpp

```diff
@@ -57,32 +63,12 @@ class FieldManager {
 
   // Change the state of the database
   void registration_begins ();
+  void register_field (const FieldIdentifier& fid) { register_field(FieldRequest(fid)); }
```

Comment:
Worth calling out that this overload assumes a pack size of 1? Just curious.

Reply:
Actually, I think I left this ctor to accommodate the use in `field_tests.cpp`. However, I'm trhinking of removing it, and force the user to build a FieldRequest (which can be done on the fly).

Reply:
Turns out, it wasn't used anywhere at all. Purged.